sdk/include/stdlib.h

```diff
@@ -4,10 +4,37 @@
 #pragma once
 
 #include <cdefs.h>
+#include <compartment-macros.h>
 #include <stddef.h>
 #include <stdint.h>
 #include <timeout.h>
 
+#ifndef MALLOC_DEFAULT_QUOTA
+#	define MALLOC_DEFAULT_QUOTA 4096
+#endif
+struct MallocCapability
```

Comment:
Hm. I think I know why you called this a `Capability`, and I don't completely disagree, but I might prefer `MallocCapabilityState` or `MallocAuthorityState` or something along those lines: it's not a CHERI capability, and (an instance of) *this structure* isn't presented as a capability would be, in general: a (sealed) capability to (a wrapper around) an instance of this structure is the actual bearer token.